server/src/main/java/org/eclipse/lsp/cobol/core/visitor/CobolVisitor.java

```diff
@@ -545,6 +546,52 @@ private NodeType getDataDescriptionNodeType(DataDescriptionEntryFormat1Context c
     return result;
   }
 
+  @Override
+  public List<Node> visitDbs_host_variable(Dbs_host_variableContext ctx) {
+    List<Node> result = new ArrayList<>();
+    addSqlValueToUsage(result, ctx);
+    result.addAll(visitChildren(ctx));
```

Comment:
It will add the children to the same level. Please, use `addTreeNode` here and in `visitDbs_rs_locator_variable`

Reply:
> It will add the children to the same level. Please, use `addTreeNode` here and in `visitDbs_rs_locator_variable`

There are many places in existing program which should be migrated to new logic (constants, .. ),  I guess Anton's change will cover all together.

Reply:
we discussed it with Andrei, suggested returning addTreeNode will not work here. Needs refactoring, so better do it for all place together.